orchagent/fdborch.cpp

```diff
@@ -174,21 +174,32 @@ void FdbOrch::update(sai_fdb_event_t type, const sai_fdb_entry_t* entry, sai_obj
                 update.add = false;
                 itr++;
 
-                storeFdbEntryState(update);
+                if (SAI_NULL_OBJECT_ID != bridge_port_id)
+                {
+                    if (!m_portsOrch->getPortByBridgePortId(bridge_port_id, update.port))
+                    {
+                        SWSS_LOG_ERROR("Failed to get port by bridge port ID 0x%" PRIx64, bridge_port_id);
+                        return;
```

Comment:
>return [](start = 24, length = 6)

Should we not return and continue flush run on remainings?

Reply:
No. Actually, the return value of the function is independent of looping items. This should move outside or for loop.